sre-recipes/sandboxctl

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+"""
+This CLI is the interface for sre recipes. It deploys and rolls back
+broken services as well as verifies that the user found the correct
+cause of the broken service.
+
+For information on how to run the CLI, run the following:
+`python3 sandboxctl --help`
+"""
+
+import logging
+import os
+from os.path import dirname, abspath
+from pkgutil import iter_modules
+from pathlib import Path
+from importlib import import_module
+from inspect import isclass
+import click
+from recipe import Recipe
+
+def get_valid_recipes(attributes):
+    """
+    Gets all valid recipes from the given list of tuples of attribute names
+    and attributes, and returns a map containing recipe names and recipe objects
+    """
+
+    recipe_objs = {}
+    recipe_num = 1
+    for attribute_name, attribute in attributes:
```

Comment:
instead of managing `recipe_num` manually, you could do:

`for recipe_num, (attribute_name, attribute) in enumerate(attributes):`

That gives you all three as part of the loop

Reply:
I didn't know that, thanks!